Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -274,7 +288,7 @@ public void Settings_LanguageMissing_NoError()
 
 <ItemGroup>
   <Analyzer Include='should.be.removed.analyzer1.dll' />
```

Comment:
[education] why should `should.be.removed.analyzer1.dll` be removed?

the question applies also to similar scenario in Settings_SettingsMissing_NoError

Reply:
Honestly I do not see why this behavior. It was there before me 😄. There is a comment in the code that says that this is the expected behavior. See: https://github.com/SonarSource/sonar-scanner-msbuild/blob/a389477c7f90a345be19789ebe420461a022bb3e/src/SonarScanner.MSBuild.Tasks/GetAnalyzerSettings.cs#L125.
If I did the correct digging the behavior was introduced by @duncanp-sonar so, he might have an answer for us?

Reply:
maybe send him an email to point to this comment when he gets back

Reply:
@csaba-sagi-sonarsource, @andrei-epure-sonarsource I think was to exactly preserve the old behaviour before the 'sonar.cs.roslyn.xxx` settings were introduced i.e. the user is using a new version of the scanner with an old version of the SonarDotNet plugin that didn't have those new arguments.